packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -41,6 +41,7 @@ type Props = {
   setInspectFrame: (inspectFrame: Frame | null) => void;
   setInspectStackData: (inspectStackData: InspectStackData | null) => void;
   onInspectorItemSelected: (item: InspectDataStackItem) => void;
+  resetInspector: () => void;
```

Comment:
There's already setIsInspecting and now resetInspector. We should only have one callback to control inspect state